content/topics/file-and-directory-naming/general/_index.md

```diff
@@ -0,0 +1,21 @@
+--- 
+content_type: topic
+ready: true
+title: File and directory naming 
+---
+
+## Naming files and directories
+
+Naming conventions are rules which enable the titling of electronic and physical folders, documents and records in a consistent and logical way.
```

Comment:
[This is worth looking at](https://www.google.com/search?q=people+who+use+overly+complicated+language&oq=people+who+use+overly+complicated+language&aqs=chrome..69i57.9519j0j7&sourceid=chrome&ie=UTF-8) 

Can you rewrite this in a way that is as simple and clear as possible. What do you want the learner to know?